### ISSUE TITLE
About the performance

### ISSUE THREAD

**Issue by Whalesong-zrs** [ChenyanWu/MEBOW#12]
I use the same setting,include number of GPUS, batch size(all the same as the yaml in the config),hoe_sigma=4,however,my result is like this  
acc5   acc15   acc22.5  acc30  acc45
68.2   89.7      93.1       96.1     97.5
and I only focus on the orientation problem so I don't use joint point.Is the random seed changes the performance?

**Comment by ChenyanWu** [comment]
Your performance is close to the paper - acc5 68.6 acc15 90.7 acc30 96.9. I think it is in the range of random noise. CUDA version,  GPU difference, Pytorch version, and random seed may all slightly influence the results. You may tune the batch size or learning rate a little bit to achieve the optimal performance in your computing environment.
